src/Classes/Orchestrator/Choria.php

```diff
@@ -40,6 +42,7 @@ class Choria implements OrchestratorInterface
     private static $dispatchCommand = 'mco playbook run helio::task::update --input \'{"cluster_address":"%s","task_ids":"[%s]"}\'';
     private static $joinWorkersCommand = 'mco playbook run helio::queue --input \'{"cluster_join_token":"%s","cluster_join_address":"%s","cluster_join_count":"%s","manager_id":"%s"}\'';
     private static $updateJobCommand = 'mco playbook run helio::job::update --input \'{"node":"%s","ids":"%s","user_id":"%s"}\'';
+    private static $serviceScaleCommand = 'mco playbook run helio::cluster::services::scale --input \'$serviceScaleArray\'';
```

Comment:
```suggestion
    private static $serviceScaleCommand = 'mco playbook run helio::cluster::services::scale --input \'{{serviceScaleArray}}\'';
```

I'd use something which is not tempting to be replaced by PHP (in case we'd switch to double quotes, for some weird reason)

Reply:
I definitely agree with you, but we should apply it for all the commands...